packages/admin/dashboard/src/routes/orders/order-detail/components/order-fulfillment-section/order-fulfillment-section.tsx

```diff
@@ -108,33 +109,78 @@ const UnfulfilledItem = ({
 }
 
 const UnfulfilledItemBreakdown = ({ order }: { order: AdminOrder }) => {
-  const { t } = useTranslation()
-
   // Create an array of order items that haven't been fulfilled or at least not fully fulfilled
-  const unfulfilledItems = order.items!.filter(
-    (i) => i.detail.fulfilled_quantity < i.quantity
+  const unfulfilledItemsWithShipping = order.items!.filter(
+    (i) => i.requires_shipping && i.detail.fulfilled_quantity < i.quantity
+  )
+
+  const unfulfilledItemsWithoutShipping = order.items!.filter(
+    (i) => !i.requires_shipping && i.detail.fulfilled_quantity < i.quantity
   )
 
-  if (!unfulfilledItems.length) {
+  if (
```

Comment:
nit: we can remove this since render will return only an empty fragment if values don't exist